ci/credentials.example.yml

```diff
@@ -0,0 +1,38 @@
+cg-quotas-db-git-url: https://github.com/18F/cg-quotas-db
+cg-quotas-db-git-staging-branch: master
+cg-quotas-db-git-prod-branch: master
+
+staging-cf-api-url: https://api.your.cf.installation
+staging-cf-deploy-username: a-user
+staging-cf-deploy-password: a-pass
+staging-cf-organization: an-org
+staging-cf-space: a-space
+
+prod-cf-api-url: https://api.your.cf.installation
```

Comment:
Minor: I would set the `api-url`s to the actual values, just to have one less thing to configure.